src/constants/data.ts

```diff
@@ -8,37 +8,32 @@ import {
   IncorrectEntriesIcon,
   AlmostCorrectEntriesIcon
 } from '../../assets/images'
-
-export const SCREENS = {
-  profession: 'Profession',
-  professionSubcategory: 'ProfessionSubcategory',
-  exercises: 'Exercises',
-  vocabularyOverview: 'VocabularyOverview',
-  vocabularyTrainer: 'VocabularyTrainer',
-  initialSummaryScreen: 'InitialSummary',
-  ResultsOverview: 'ResultsOverview',
-  CorrectResults: 'CorrectResults',
-  ResultScreen: 'ResultScreen',
-  AlmostCorrectResults: 'AlmostCorrectResults',
-  IncorrectResults: 'IncorrectResults'
-}
+import { RoutesParamsType } from '../navigation/NavigationTypes'
 
 export const RESULT_TYPE = ['correct', 'similar', 'incorrect']
```

Comment:
Can you refactor this. I think this should be a union type and be used correctly then?

Reply:
See other PRs, did some more refactoring there, maybe that is already enough.